src/sentry/web/frontend/openidtoken.py

```diff
@@ -47,6 +48,24 @@ def get_encrypted_id_token(self):
             "exp": self.exp,
             "iat": self.iat,
         }
+        user_details = self._get_user_details(grant)
+        claims.update(user_details)
         if self.nonce:
             claims["nonce"] = self.nonce
         return jwt_utils.encode(claims, self.shared_secret, headers={**headers, "alg": "HS256"})
+
+    def _get_user_details(self, grant):
+        user_details = {}
+        if grant.has_scope("profile"):
+            profile_details = {
+                "name": grant.user.name,
+                "avatar_type": grant.user.avatar_type,
+                "avatar_url": grant.user.avatar_url,
+                "date_joined": str(grant.user.date_joined),
+            }
+            user_details.update(profile_details)
+        if grant.has_scope("email"):
+            email = UserEmail.objects.get(user=grant.user)
+            email_details = {"email": email.email, "email_verified": email.is_verified}
```

Comment:
Users can have *many* email addresses. Using `.get()` here will fail for lots of folks. You should have a list of email addresses, or only show the 'primary' email address which is in user.email.

Reply:
Good catch! Didn't think about this - fixed it